src/components/PathwayExercise/ContentEdit/styles.scss

```
@@ -0,0 +1,31 @@
+.embed-tool__caption {
+  display: none;
+}
+
```

When you need to use a hack to make something work, it might be useful to document it in the issues and then reference it in a comment. This way we can keep track of them. Doing so will also allow a potential contributor to search the issues and perhaps make a PR if they think of a better solution that comes up in the future.
 
```suggestion

// Issue # 811: Hack to remove image caption
```